tests/PopularDishList.test.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { shallow, mount, render } from 'enzyme';
+import PopularDishList from '../client/src/Components/PopularDishList.jsx'
+import PictureList from '../client/src/Components/PictureList.jsx'
+
+describe('PopularDishList', () => {
+
+  it('should check if prop exist', () => {
```

Comment:
this isn't really a test for popularDish, it's just testing if PictureList exists on its own. You could use wrapper.find(PictureList), which will return an array of the PictureList components rendered by PopularDishList, and test if the length of that array equals the number of PictureList components you expected PopularDishList to render